### PR TITLE
Update Mac installation instructions

### PR DESCRIPTION
**Pull request by mbovel**
- Remove specialized message for Apple Silicon.
- Use the `coursier` brew formula.

### CODE REVIEW

File: _ja/overviews/macros/usecases.md
Comment (hamzaremmal) [inline _ja/overviews/macros/usecases.md:25]: These are being fixed by another PR (https://github.com/scala/docs.scala-lang/pull/3058). Maybe @SethTisue you can extract them into another PR and have them merged asap ?

**Review by hamzaremmal — APPROVED** [review]

**Comment by hamzaremmal** [comment]
I will wait for @SethTisue's approval before merging

File: _ja/overviews/macros/usecases.md
Comment (mbovel) [reply _ja/overviews/macros/usecases.md:25]: I might a better fix though: I updated the links instead of disabling them :)

File: _data/setup-scala.yml
Comment (SethTisue) [inline _data/setup-scala.yml:5]: Why the change from `cs` to `coursier`? I've never been clear on the difference. I think I usually see `cs`?

File: _ja/overviews/macros/usecases.md
Comment (SethTisue) [reply _ja/overviews/macros/usecases.md:25]: Yeah Matt's fix is better.

File: _data/setup-scala.yml
Comment (mbovel) [reply _data/setup-scala.yml:5]: Because for me, `brew install coursier` only installs `coursier`. It's then `coursier setup` that installs other things such as `cs`:

```
➜  ~ brew install coursier
...
➜  ~ which coursier   
/usr/local/bin/coursier
➜  ~ which cs
cs not found
➜  ~ coursier setup
...
Checking if the standard Scala applications are installed
  Installed ammonite
  Installed cs
  Installed coursier
  Installed scala
  Installed scalac
  Installed scala-cli
  Installed sbt
  Installed sbtn
```

Comment (bishabosha) [reply _data/setup-scala.yml:5]: There's a difference between the homebrew-core formula and couriers own tap - core doesn't include cs but the tap does

Comment (bishabosha) [reply _data/setup-scala.yml:5]: but seeing as the tap doesn't have an option for apple silicon this is better

Comment (bishabosha) [reply _data/setup-scala.yml:5]: @SethTisue the difference is that coursier is a jar-based launcher rather than graalvm native image

File: _ja/overviews/macros/usecases.md
Comment (hamzaremmal) [reply _ja/overviews/macros/usecases.md:25]: Agreed

File: _data/setup-scala.yml
Comment (SethTisue) [reply _data/setup-scala.yml:5]: okay, I mean at some level this is all deeply confusing and clear as mud, but it sounds like this is mergeable :shrug:

**Pull request merged**
